README.md

```diff
@@ -6,20 +6,73 @@ This is a Kubernetes Operator (https://coreos.com/operators/) which deploys Mong
 
 This codebase is currently _pre-alpha_, and is not ready for use.
 
-If you are a MongoDB Enterprise customer, or need Enterprise features such as Backup, you may be looking for the [MongoDB Enterprise Operator for Kubernetes](https://github.com/mongodb/mongodb-enterprise-kubernetes).
+If you are a MongoDB Enterprise customer, or need Enterprise features such as Backup, you can use the [MongoDB Enterprise Operator for Kubernetes](https://github.com/mongodb/mongodb-enterprise-kubernetes).
 
+## Installation
 
+### Prerequisites
 
+Before you install the MongoDB Community Kubernetes Operator, you must:
 
-## Licensing
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+2. Install [kind](https://kind.sigs.k8s.io/)
+3. Clone this repository.
+   ```
+   git clone https://github.com/mongodb/mongodb-kubernetes-operator.git
+   ```
 
-The source code of this Operator is available under the Apache v2 license.
+### Installing the MongoDB Community Kubernetes Operator
 
-The MongoDB Agent binary in the agent/ directory may be used under the "Free for Commercial Use" license found in agent/LICENSE.
+The MongoDB Community Kubernetes Operator is a collection of [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and a controller.
```

Comment:
as of right now it is just a single CRD.

Reply:
Fixed.